package.json

```
@@ -1172,6 +1177,11 @@
                 "category": "AWS",
                 "icon": "$(new-folder)"
             },
+            {
+                "command": "aws.s3.addBucket",
```

for consistency with `aws.showRegion` ?
```suggestion
                "command": "aws.s3.showBucket",
```